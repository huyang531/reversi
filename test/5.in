cd
cc
dc
ec
fd
ce
eb
fe
ff
fa
ef
fb
cf
df
be
bd
dg
af
bc
ac
cb
db
da
gd
ea
fc
ge
ca
hd
eg
cg
dh
ch
bh
ad
ae
eh
fh
bf
gc
fg
gh
bg
hc
hb
hf
gf
gg
ba
aa
ga
ha
hh
he
gb
hg
ah
bb
ab
ag
