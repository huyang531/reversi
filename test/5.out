  abcdefgh
a ........
b ........
c ........
d ...WB...
e ...BW...
f ........
g ........
h ........
Enter move for colour B (RowCol):   abcdefgh
a ........
b ........
c ...B....
d ...BB...
e ...BW...
f ........
g ........
h ........
Enter move for colour W (RowCol):   abcdefgh
a ........
b ........
c ..WB....
d ...WB...
e ...BW...
f ........
g ........
h ........
Enter move for colour B (RowCol):   abcdefgh
a ........
b ........
c ..WB....
d ..BBB...
e ...BW...
f ........
g ........
h ........
Enter move for colour W (RowCol):   abcdefgh
a ........
b ........
c ..WB....
d ..WBB...
e ..WWW...
f ........
g ........
h ........
Enter move for colour B (RowCol):   abcdefgh
a ........
b ........
c ..WB....
d ..WBB...
e ..WBW...
f ...B....
g ........
h ........
Enter move for colour W (RowCol):   abcdefgh
a ........
b ........
c ..WWW...
d ..WWW...
e ..WBW...
f ...B....
g ........
h ........
Enter move for colour B (RowCol):   abcdefgh
a ........
b ........
c ..WWW...
d ..WWW...
e .BBBW...
f ...B....
g ........
h ........
Enter move for colour W (RowCol):   abcdefgh
a ........
b ........
c ..WWW...
d ..WWW...
e .BBWW...
f ...BW...
g ........
h ........
Enter move for colour B (RowCol):   abcdefgh
a ........
b ........
c ..WWW...
d ..WWW...
e .BBWW...
f ...BBB..
g ........
h ........
Enter move for colour W (RowCol):   abcdefgh
a ........
b ........
c ..WWW...
d ..WWW...
e .WBWW...
f W..BBB..
g ........
h ........
Enter move for colour B (RowCol):   abcdefgh
a ........
b ........
c ..WWW...
d ..WWW...
e .WBBBB..
f W..BBB..
g ........
h ........
Enter move for colour W (RowCol):   abcdefgh
a ........
b ........
c ..WWW...
d ..WWW...
e .WWBBB..
f WW.BBB..
g ........
h ........
Enter move for colour B (RowCol):   abcdefgh
a ........
b ........
c ..WWWB..
d ..WWB...
e .WWBBB..
f WW.BBB..
g ........
h ........
Enter move for colour W (RowCol):   abcdefgh
a ........
b ........
c ..WWWB..
d ..WWWW..
e .WWBBB..
f WW.BBB..
g ........
h ........
Enter move for colour B (RowCol):   abcdefgh
a ........
b ....B...
c ..WWBB..
d ..WWBW..
e .WWBBB..
f WW.BBB..
g ........
h ........
Enter move for colour W (RowCol):   abcdefgh
a ........
b ...WB...
c ..WWWB..
d ..WWBW..
e .WWBBB..
f WW.BBB..
g ........
h ........
Enter move for colour B (RowCol):   abcdefgh
a ........
b ...WB...
c ..WWWB..
d ..WWBBB.
e .WWBBB..
f WW.BBB..
g ........
h ........
Enter move for colour W (RowCol):   abcdefgh
a .....W..
b ...WW...
c ..WWWB..
d ..WWBBB.
e .WWBBB..
f WW.BBB..
g ........
h ........
Enter move for colour B (RowCol):   abcdefgh
a .....W..
b ..BWW...
c ..WBWB..
d ..WWBBB.
e .WWBBB..
f WW.BBB..
g ........
h ........
Enter move for colour W (RowCol):   abcdefgh
a ..W..W..
b ..WWW...
c ..WBWB..
d ..WWBBB.
e .WWBBB..
f WW.BBB..
g ........
h ........
Enter move for colour B (RowCol):   abcdefgh
a ..W..W..
b ..WWW...
c .BBBWB..
d ..BWBBB.
e .WWBBB..
f WW.BBB..
g ........
h ........
Enter move for colour W (RowCol):   abcdefgh
a ..W..W..
b ..WWW...
c .BWBWB..
d .WWWBBB.
e .WWBBB..
f WW.BBB..
g ........
h ........
Enter move for colour B (RowCol):   abcdefgh
a ..W..W..
b ..WWW...
c .BWBWB..
d BBBBBBB.
e .WWBBB..
f WW.BBB..
g ........
h ........
Enter move for colour W (RowCol):   abcdefgh
a ..W..W..
b ..WWW...
c .BWWWB..
d BBBWBBB.
e .WWWBB..
f WW.WBB..
g ...W....
h ........
Enter move for colour B (RowCol):   abcdefgh
a ..W..W..
b ..WWW...
c .BWWWB..
d BBBWBBB.
e BBBBBB..
f WW.WBB..
g ...W....
h ........
Enter move for colour W (RowCol):   abcdefgh
a ..W..W..
b ..WWW...
c .BWWWB..
d BBWWBBB.
e BBWBBB..
f WWWWBB..
g ...W....
h ........
Enter move for colour B (RowCol):   abcdefgh
a ..W..W..
b ..WWW...
c .BWWWB..
d BBWWBBB.
e BBBBBB..
f WWWBBB..
g ...WB...
h ........
Enter move for colour W (RowCol):   abcdefgh
a ..W..W..
b ..WWW...
c WWWWWB..
d WBWWBBB.
e WBBBBB..
f WWWBBB..
g ...WB...
h ........
Enter move for colour B (RowCol):   abcdefgh
a ..W..W..
b ..WWW...
c WWWWWB..
d WBWWBBB.
e WBBBBB..
f WWWBBB..
g ...BB...
h ...B....
Enter move for colour W (RowCol):   abcdefgh
a ..W..W..
b ..WWW...
c WWWWWB..
d WBWWBWB.
e WWWWWWW.
f WWWBBB..
g ...BB...
h ...B....
Enter move for colour B (RowCol):   abcdefgh
a ..W..W..
b ..WWW...
c WWWWWBB.
d WBWWBBB.
e WWWWBWW.
f WWWBBB..
g ...BB...
h ...B....
Enter move for colour W (RowCol):   abcdefgh
a ..W..W..
b ..WWW...
c WWWWWBB.
d WBWWWWWW
e WWWWBWW.
f WWWBBB..
g ...BB...
h ...B....
Enter move for colour B (RowCol):   abcdefgh
a ..W..W..
b ..WWW...
c WWWWWBBB
d WBWWWWBW
e WWWWBBW.
f WWWBBB..
g ...BB...
h ...B....
Enter move for colour W (RowCol):   abcdefgh
a ..W..W..
b ..WWW..W
c WWWWWBWW
d WBWWWWBW
e WWWWBBW.
f WWWBBB..
g ...BB...
h ...B....
Enter move for colour B (RowCol):   abcdefgh
a ..WB.W..
b ..WBB..W
c WWWBWBWW
d WBWBWWBW
e WWWBBBW.
f WWWBBB..
g ...BB...
h ...B....
Enter move for colour W (RowCol):   abcdefgh
a ..WWWW..
b ..WWW..W
c WWWBWBWW
d WBWBWWBW
e WWWBBBW.
f WWWBBB..
g ...BB...
h ...B....
Enter move for colour B (RowCol):   abcdefgh
a ..WWWW..
b ..WWW..W
c WWWBWBWW
d WBWBWWBW
e WWWBBBBB
f WWWBBB..
g ...BB...
h ...B....
Enter move for colour W (RowCol):   abcdefgh
a ..WWWW..
b ..WWW..W
c WWWBWBWW
d WBWBWWBW
e WWWBBBWW
f WWWBBB.W
g ...BB...
h ...B....
Enter move for colour B (RowCol):   abcdefgh
a ..WWWW..
b ..WWWB.W
c WWWBBBWW
d WBWBWWBW
e WWWBBBWW
f WWWBBB.W
g ...BB...
h ...B....
Enter move for colour W (RowCol):   abcdefgh
a ..WWWW..
b ..WWWB.W
c WWWBBBWW
d WBWBWWBW
e WWWBWBWW
f WWWWBB.W
g ..WBB...
h ...B....
Enter move for colour B (RowCol):   abcdefgh
a ..WWWW..
b ..WWWB.W
c WWWBBBWW
d WBWBWWBW
e WWWBWBBW
f WWWWBBBW
g ..WBB...
h ...B....
Enter move for colour W (RowCol):   abcdefgh
a ..WWWW..
b ..WWWB.W
c WWWBBBWW
d WBWBWWBW
e WWWBWWBW
f WWWWBBWW
g ..WBB..W
h ...B....
Enter move for colour B (RowCol):   abcdefgh
a ..WWWW..
b ..WWWBBW
c WWWBBBBW
d WBWBWWBW
e WWWBWWBW
f WWWWBBWW
g ..WBB..W
h ...B....
Enter move for colour W (RowCol):   abcdefgh
a ..WWWW..
b ..WWWBBW
c WWWBBBBW
d WBWBWWBW
e WWWBWWBW
f WWWWWBWW
g ..WWB..W
h ..WB....
Enter move for colour B (RowCol):   abcdefgh
a ..WWWW..
b ..WWWBBW
c WWWBBBBW
d WBWBWBBW
e WWWBBWBW
f WWWBWBWW
g ..BWB..W
h .BBB....
Enter move for colour W (RowCol):   abcdefgh
a ..WWWW..
b ..WWWBBW
c WWWBBBBW
d WBWBWBBW
e WWWBBWBW
f WWWWWBWW
g ..BWW..W
h .BBB.W..
Enter move for colour B (RowCol):   abcdefgh
a ..WWWW..
b ..WWWBBW
c WWWBBBBW
d WBWBWBBW
e WWWBBWBW
f WWWWBBWW
g ..BBBB.W
h .BBB.W..
Enter move for colour W (RowCol):   abcdefgh
a ..WWWW..
b ..WWWBBW
c WWWBBBBW
d WBWWWBBW
e WWWBWWBW
f WWWWBWWW
g ..BBBBWW
h .BBB.W..
Enter move for colour B (RowCol):   abcdefgh
a ..WWWW..
b B.WWWBBW
c WBWBBBBW
d WBBWWBBW
e WWWBWWBW
f WWWWBWWW
g ..BBBBWW
h .BBB.W..
Enter move for colour W (RowCol):   abcdefgh
a W.WWWW..
b W.WWWBBW
c WBWBBBBW
d WBBWWBBW
e WWWBWWBW
f WWWWBWWW
g ..BBBBWW
h .BBB.W..
Enter move for colour B (RowCol):   abcdefgh
a W.WWWW..
b W.WWWBBW
c WBWBBBBW
d WBBBWBBW
e WWBBWWBW
f WBWWBWWW
g B.BBBBWW
h .BBB.W..
Enter move for colour W (RowCol):   abcdefgh
a W.WWWW..
b W.WWWBBW
c WBWBBBBW
d WBBBWBBW
e WWBBWWBW
f WBWWBWWW
g W.BBBBWW
h WBBB.W..
Enter move for colour B (RowCol):   abcdefgh
a W.WWWW..
b W.WWWBBW
c WBWBBBBW
d WBBBWBBW
e WWBBBWBW
f WBWWBBWW
g W.BBBBBW
h WBBB.W.B
Enter move for colour W (RowCol):   abcdefgh
a W.WWWW..
b W.WWWBBW
c WBWBBBBW
d WBBBWBBW
e WWBBWWBW
f WBWWWBWW
g W.BWWWBW
h WWWWWW.B
Enter move for colour B (RowCol):   abcdefgh
a W.WWWW..
b W.WWWBBW
c WBWBBBBW
d WBBBWBBW
e WWBBWWBW
f WBBWWBWW
g WBBWWWBW
h WWWWWW.B
Enter move for colour W (RowCol):   abcdefgh
a W.WWWW..
b W.WWWBBW
c WBWBBBBW
d WBBBWBBW
e WWBBWWBW
f WBBWWBWW
g WBBWWWWW
h WWWWWWWB
Enter move for colour B (RowCol):   abcdefgh
a W.WWWW.B
b W.WWWBBB
c WBWBBBBB
d WBBBWBBB
e WWBBWWBB
f WBBWWBWB
g WBBWWWWB
h WWWWWWWB
Enter move for colour W (RowCol):   abcdefgh
a W.WWWW.B
b WWWWWBBB
c WWWBBBBB
d WWBBWBBB
e WWBBWWBB
f WBBWWBWB
g WBBWWWWB
h WWWWWWWB
Enter move for colour B (RowCol):   abcdefgh
a WBWWWW.B
b WBBWWBBB
c WBWBBBBB
d WBBBWBBB
e WBBBWWBB
f WBBWWBWB
g WBBWWWWB
h WWWWWWWB
Enter move for colour W (RowCol):   abcdefgh
a WBWWWWWB
b WBBWWWWB
c WBWBWBWB
d WBBWWBWB
e WBWBWWWB
f WWBWWBWB
g WBBWWWWB
h WWWWWWWB
B player has no valid move.
W player has no valid move.
White wins by 18 points!
